reactComponents/src/searchbar/icons/zero.exchange.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC, memo } from 'react';
+import { IIconProps } from '.';
+
+const ZeroExchangeIcon: FC<IIconProps> = memo(
+  ({ active, activeColor, color, height, width }) => (
+    <svg
```

Comment:
These look to be the defaults for all icons. Should they be abstracted into a new component?

Reply:
could you explain this a bit more?

Reply:
All these SVGs are rendered with width, height, fill, and viewbox which seem to be the same for each icon. Could this be abstracted into an another component such as SvgIcon that would default to these?

```
const SvgIcon = ({ children, ... }) => (
	<svg width={...} ...>
		{children}
	</svg>
)
```